_data/apps.yml

```diff
@@ -1035,3 +1035,18 @@
     - "research"
     - "spreadsheet"
   icon: "dataproofer.png"
+-
+  name: "PupaFM"
+  description: "douban.fm Music Desktop Player"
+  website: "https://github.com/xwartz/PupaFM"
+  keywords:
+    - "douban.fm"
+    - "fm"
+    - "douban"
+    - "PupaFM"
+    - "music player"
+    - "react"
+    - "redux"
+  license: "MIT"
+  icon: "pupafm.png"
+  featuredOnHomepage: true
```

Comment:
Please remove this line, we aren't currently adding new homepage icons at this time.